static/src/js/components/TemporalSelection/TemporalSelection.jsx

```diff
@@ -227,7 +227,7 @@ export class TemporalSelection extends Component {
               controlId={`${controlId}__recurring`}
             >
               <Form.Check>
-                <Form.Check.Input disabled={enableRecurring} type="checkbox" onChange={onRecurringToggle} checked={isRecurring} />
+                <Form.Check.Input aria-label="is-recurring-checkbox" disabled={enableRecurring} type="checkbox" onChange={onRecurringToggle} checked={isRecurring} />
```

Comment:
This is a dev artifact while converting to RTL we should either use it in the enzyme test or not